test/libsolidity/semanticTests/using/imported_functions.sol

```diff
@@ -0,0 +1,19 @@
+==== Source: A ====
+function inc(uint x) pure returns (uint) {
+    return x + 1;
+}
+
+==== Source: B ====
+import {inc as f} from "A";
+import "A";
+using * for uint;
```

Comment:
This is neither in the grammar nor in the documentation, is it?
Even more so, is this really something we expect to be used :-)?